src/collector/Backend.cpp

```diff
@@ -54,8 +53,6 @@ void Backend::clone_from(const Backend & other)
 
     std::memcpy(&m_stat, &other.m_stat, sizeof(m_stat));
     std::memcpy(&m_calculated, &other.m_calculated, sizeof(m_calculated));
```

Comment:
`m_fs` and `m_group` are set in `merge_*` functions. Objects pointed to by these pointers are owned by storage.